manifests/sysctl.pp

```diff
@@ -6,13 +6,11 @@
     schedule => $sys_schedule,
   }
   if $config {
-    include augeasproviders
     create_resources(sysctl, $config, $defaults)
   }
   else {
     $hiera_config = hiera_hash('system::sysctl', undef)
     if $hiera_config {
-      include augeasproviders
```

Comment:
This looks correct as there is no such class.

Reply:
It was removed *ages* ago. https://github.com/hercules-team/augeasproviders/commit/82e4a62312a9ba8b0aaaebada7b600b70159abb7#diff-60ae41fd0a31977447947f59940ee9a4

Reply:
Could you also remove the `include augeasproviders` from providers.pp?